tests/backstop_data/engine_scripts/puppet/activity-feed-panel/bulk-actions.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const Utility = require('./../utility.js');
+
+module.exports = async (page, scenario, vp) => {
+  const utility = new Utility(page, scenario, vp);
+
+  await utility.waitForAngular();
+  await utility.waitForLoadingComplete();
+  await page.click('.civicase__bulkactions-checkbox-toggle');
+  await page.waitFor(300);
```

Comment:
We should not have arbitrary waits. We should wait for specific elements(tobe hidden/visible or something else).

Reply:
@deb1990 the element is visible, the 300 is for the animation. Otherwise the screenshot will be taken in the middle of an animation instead of the final state.

Reply:
ok makes sense. please mention this in a inline comment also, otherwise it looks like a arbitrary wait.